### PR TITLE
[CPU] Switch to the original oneDNN PReLU post op

### PR DESCRIPTION
**Pull request by maxnick**
### Details:
In this PR the original oneDNN PReLU postop is employed instead of the custom "bynary prelu" developed by the OV team. The old "bynary prelu" is removed. It's been done in order to fix a bug and reduce the fork size simultaneously.
MatMul and FC single layer tests were extended with negative input values in order to actually test ReLU and its derivative post ops.

oneDNN fork PR: https://github.com/openvinotoolkit/oneDNN/pull/241

### Tickets:
 - CVS-137492

### CODE REVIEW

**Comment by maxnick** [comment]
@luweizhou2016 , FIY

File: src/plugins/intel_cpu/src/dnnl_postops_composer_legacy.cpp [hunk 1]
```diff
@@ -95,6 +95,24 @@ void DnnlPostOpsComposerLegacy::appendBinary(const dnnl::algorithm alg, const st
     args[DNNL_ARG_ATTR_MULTIPLE_POST_OP(ops.len() - 1) | DNNL_ARG_SRC_1] = mem;
 }
 
+void DnnlPostOpsComposerLegacy::appendPReLU(const std::vector<float>& data) {
```
Comment (luweizhou2016) [inline src/plugins/intel_cpu/src/dnnl_postops_composer_legacy.cpp:98]: May I know what is the difference between class DnnlPostOpsComposerLegacy and DnnlPostOpsCompose. I can check only fc use DnnlPostOpsCompose. Do we have plan to remove DnnlPostOpsComposeLegacy for long term or will keep 2 both in the code.

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 2]
```diff
@@ -285,6 +285,19 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*smoke_RDFT_CPU_4D/RDFTTestCPU.CompareWithRefs/prec=f32_IS0=\[\?.192.36.64\]_.*_axes=\(\((0|_2._1|_1|1)\)_.*isInverse=false.*)",
         R"(.*smoke_RDFT_CPU_4D/RDFTTestCPU.CompareWithRefs/prec=f32_IS0=\[\]_TS0=\(\((1.120.64.64|1.120.96.96|\?.\?.\?.\?|1.192.\?.\?|1..2.\?.\?.1..100)\)\).*isInverse=false.*)",
         R"(.*smoke_RDFT_2d/RDFTLayerTest.Inference/IS=\(100.16\)_modelType=f32_Axes=\((0.1|_2._1|1.0)\)_SignalSize=\(\).*)",
+
```
Comment (luweizhou2016) [inline src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp:288]: I am curious why jit gemm with prelu as post ops can pass the test. After this change, these tests should fail in my understanding. Did I miss something?

Comment (maxnick) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp]: I added PReLU per tensor post of to the list of fusing operations in the conv SL tests.

**Review by luweizhou2016 — APPROVED** [review]
LTGM. CI needs green. Also some acc test on avx2 maybe on demand to double check because the code path changes.

**Comment by maxnick** [comment]
@dmitry-gorokhov , could you please take a look?

**Pull request closed without merging**